packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -28,6 +28,7 @@ export interface PanelChromeProps {
   height: number;
   children: (innerWidth: number, innerHeight: number) => ReactNode;
   padding?: PanelPadding;
+  transparent?: boolean;
```

Comment:
@polibb ah, was hoping this was going to be added in a separate PR. 

I was thinking of a property named background with options `transparent` | 'primary'   (default to primary) is preferrable. I can see a future where we want to support different backgrounds (potentially panels with secondary background, or some other background)

Reply:
@polibb 

or maybe background is not the best name as it also controls border.

displayMode?: 'default' | 'transparent' ; 

naming is hard :)

Reply:
@torkelo I see, this is better off in a separate PR then. The only reason I added it is because I thought that's the only case we're covering and it's an easy change. But in a separate PR we can think about the potential use cases.